Makefile

```diff
@@ -23,6 +23,17 @@ go:
 	@echo "Generating go protobuf files"
 	docker run --pull always --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -d protos -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
 
+python: env/pyenv.cfg
```

Comment:
Can you add `python` to the `all:` target?

Reply:
[ffa2192](https://github.com/sigstore/protobuf-specs/pull/25/commits/ffa2192f5cc476fff26ef0f26f5fec413c0b17a5)